src/wfs/src/main/java/org/geoserver/wfs/GetFeature.java

```diff
@@ -251,8 +251,8 @@ public FeatureCollectionResponse run(GetFeatureRequest request)
         }
         
         // WFS 2.0 validation, with locks "hits" is not allowed
-        if (WFSInfo.Version.V_20.compareTo(request.getVersion()) >= 0
-                && request.isLockRequest() && request.isResultTypeHits()) {
+        final boolean isV_20 = WFSInfo.Version.V_20.compareTo(request.getVersion()) >= 0;
```

Comment:
We usually use longer local variable names, using '_' for constants.

```
final boolean WFS_2.0_SEMANTICS = WFSInfo.Version.V_20.compareTo(request.getVersion()) >= 0;
```

GeoTools [Version|http://docs.geotools.org/stable/javadocs/org/geotools/util/Version.html] also has a compareTo( version, limit) which may be appropriate depending on the new WFS 1.3.0 expectations.